index.html

```diff
@@ -1,62 +1,94 @@
-<!DOCTYPE html>
 <html>
 
 <head>
-    <title>Worlds Best Calculator</title>
-
-    <!-- CSS only -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-
     <style>
+        * {
+            margin: 0;
+            padding: 0;
+        }
 
-    </style>
+        .button {
+            width: 50;
+            height: 50;
+            font-size: 30;
+            margin: 5;
+            cursor: pointer;
+        }
 
-    <script>
-        var calculator = {
-            addNumbers: function () {
-                // hint: these are strings
-                var numberOne = document.forms.inputs.numberOne.value;
-                var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
-            },
-            subtractNumbers: function () {
-                
-            }
+        .textview {
+            width: 240;
+            margin: 3;
+            font-size: 40;
+            padding: 3;
         }
+    </style>
+
 
-    </script>
 
 </head>
 
 <body>
-    <div>Header</div>
-    <div>
-       <!--
-         - allow two numbers to be entered
-         - show buttons for add and subtract
-         - show the result
-        -->
-        <form name="inputs">
-            <div>
-                Number 1:
-                <input type="number" name="numberOne" class="form-control" placeholder="First number" aria-label="Number1" >
-            </div>
-            <div>
-                Number 2:
-                <input type="number" class="form-control" name="numberTwo">
-            </div>
-            <div>
-                <button type="button" class="btn btn-primary" onclick="calculator.addNumbers()" >Add</button>
-                <button type="button" class="btn btn-primary">Substract</button>
-            </div>
-            <div>
-                <input name="result" value="0" class="form-control" disabled />
-            </div>
+    <div class="main">
+        <form name="form">
+            <input class="textview" name="textview">
         </form>
+        <table>
```

Comment:
These number buttons are not needed for this assignment. We want to have the input fields for users to type in the number values as we started in the template in class and the repo we pulled from originally.